ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/committer/PartyAllocationCommitter.scala

```diff
@@ -0,0 +1,135 @@
+// Copyright (c) 2019 The DAML Authors. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.ledger.participant.state.kvutils.committer
+
+import com.codahale.metrics.Counter
+import com.daml.ledger.participant.state.kvutils.Conversions.buildTimestamp
+import com.daml.ledger.participant.state.kvutils.DamlKvutils._
+import com.daml.ledger.participant.state.kvutils.Pretty
+import com.digitalasset.daml.lf.data.Ref
+
+private[kvutils] case object PartyAllocationCommitter
+    extends Committer[DamlPartyAllocationEntry, DamlPartyAllocationEntry.Builder] {
+
+  private object Metrics {
```

Comment:
just heads up: I want the metrics registry to be passed from outside once we refactor the whole API. We'd allocate one "top-level" committer that gets the engine, metrics registry etc. things passed to it.